src/main/java/org/thoughtcrime/securesms/notifications/NotificationCenter.java

```diff
@@ -347,7 +347,14 @@ public void notifyMessage(int accountId, int chatId, int msgId) {
         }
         String tickerLine = shortLine;
         if (!dcChat.isMultiUser() && privacy.isDisplayContact()) {
-          tickerLine = dcMsg.getSenderName(dcContext.getContact(dcMsg.getFromId()), false) + ": " + tickerLine;
+          DcContact contact = dcContext.getContact(dcMsg.getFromId());
+          tickerLine = dcMsg.getSenderName(contact, false) + ": " + tickerLine;
+
+          if (!Util.equals(dcChat.getName(), contact.getDisplayName())) {
```

Comment:
this sounds hacky, I would directly check:
`IF dcChat is multi-user OR dcMsg has overriden sender name THEN show it ELSE don't show it